BoxContentSDK/BoxContentSDK/Requests/BOXSearchRequest.m

```diff
@@ -159,6 +159,20 @@ - (void)performRequestWithCompletion:(BOXItemArrayCompletionBlock)completionBloc
     }
 }
 
+- (void)performRequestWithCached:(BOXItemArrayCompletionBlock)cacheBlock
+                       refreshed:(BOXItemArrayCompletionBlock)refreshBlock
+{
+    if (cacheBlock) {
+        if ([self.cacheClient respondsToSelector:@selector(retrieveCacheForFolderRequest:completion:)]) {
+            [self.cacheClient retrieveCacheForSearchRequest:self completionBlock:cacheBlock];
+        } else {
+            cacheBlock(nil, 0, NSMakeRange(0, 0), nil);
```

Comment:
yes, that's what we're doing in all the requests. This gives users a chance to handle cases of no cached data and some cached data in their cache block accordingly.